mixer/cmd/root.go

```diff
@@ -105,6 +112,17 @@ var RootCmd = &cobra.Command{
 		// If so: inform, stage, and exit.
 		// If not: run command in container and cancel pre-run
 		if !cmdContains(cmd, "format-bump") && !cmdContains(cmd, "upstream-format") && cmdContains(cmd, "build") {
+			// --offline=true AND --native=false, try to see if container exists
+			if builder.Offline && !builder.Native {
+				if err := runCommand("docker", "image", "inspect", "clearlinux/mixer:"+b.State.Mix.Format); err != nil {
```

Comment:
You might want to consider putting this check inside `RunCommandInContainer` itself.

As is, it's within a code block that only runs for non-format-bump commands. But what happens if someone tries to run a format bump command with `--offline`? Technically if they already have the right containers cached, they should be able to... Is there a check somewhere that stops them?

Interesting thought: technically speaking, `mixer build format-bump` (the _downstream format bump_ command) could have a `--native` version. As long as your host tooling is the right version for the format you're building, we could call in to the build libs directly in the right order, rather than calling out to `RunCommandInContainer`...